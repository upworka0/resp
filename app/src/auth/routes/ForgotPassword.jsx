import React, { PureComponent } from 'react';
import {
  Card, Row, Col, CardBody, CardHeader,
} from 'reactstrap';

import { ForgotPasswordForm } from '../components';

export default class ForgotPassword extends PureComponent {
  constructor(props) {
    super(props);

    this.onSubmit = this.onSubmit.bind(this);
  }

  onSubmit(data) {
    // eslint-disable-next-line
    console.log('Reset Password', data);
  }

  render() {
    return (
      <section className="d-flex flex-grow-1 flex-column align-items-center justify-content-center">
        <div className="container">
          <Row noGutters>
            <Col
              md={{
                size: 8,
                offset: 2,
              }}
              lg={{
                size: 6,
                offset: 3,
              }}
            >
              <h1 className="text-center mb-3">Loji</h1>
              <Card className="border-0 shadow-z4">
                <CardHeader className="bg-primary p-1" />
                <CardBody>
                  <ForgotPasswordForm onSubmit={this.onSubmit} />
                </CardBody>
              </Card>
            </Col>
          </Row>
        </div>
      </section>
    );
  }
}
