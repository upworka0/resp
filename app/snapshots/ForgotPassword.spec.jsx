import React from 'react';
import { shallow } from 'enzyme';

import ForgotPassword from '../src/auth/routes/ForgotPassword';

describe('Forgot Password UI', () => {
  it('should render forgot password form', () => {
    const wrapper = shallow(<ForgotPassword />);
    expect(wrapper).toMatchSnapshot();
  });
});
