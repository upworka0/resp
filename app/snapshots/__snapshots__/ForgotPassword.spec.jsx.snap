// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forgot Password UI should render forgot password form 1`] = `
<section
  className="d-flex flex-grow-1 flex-column align-items-center justify-content-center"
>
  <div
    className="container"
  >
    <Row
      noGutters={true}
      tag="div"
    >
      <Col
        lg={
          Object {
            "offset": 3,
            "size": 6,
          }
        }
        md={
          Object {
            "offset": 2,
            "size": 8,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <h1
          className="text-center mb-3"
        >
          Loji
        </h1>
        <Card
          className="border-0 shadow-z4"
          tag="div"
        >
          <CardHeader
            className="bg-primary p-1"
            tag="div"
          />
          <CardBody
            tag="div"
          >
            <ForgotPasswordForm
              onSubmit={[Function]}
            />
          </CardBody>
        </Card>
      </Col>
    </Row>
  </div>
</section>
`;
